### PR TITLE
fix: hussein edge-case coverage

### PR DESCRIPTION
**Pull request by husshegz**

### CODE REVIEW

File: src/index.test.ts [hunk 1]
```diff
@@ -24,6 +24,10 @@ test("is Shannon equal to Chanon", () => {
   expect(isChanon("Shannon")).toBe(false);
 });
 
+test("is Shannon equal to Chanon", () => {
```
Comment (chanonroy) [inline src/index.test.ts:27]: "is Hussein equal to Chanon"

Comment (husshegz) [reply src/index.test.ts]: Sorry wrote this test while driving a car
